packages/turf-rectangle-grid/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@turf/rectangle-grid",
+  "version": "6.0.2",
+  "description": "turf rectangle-grid module",
+  "main": "main.js",
+  "module": "main.es.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts",
+    "main.js",
+    "main.es.js"
+  ],
+  "scripts": {
+    "pretest": "rollup -c ../../rollup.config.js",
+    "test": "node -r @std/esm test.js",
+    "posttest": "node -r @std/esm ../../scripts/validate-es5-dependencies.js",
+    "bench": "node -r @std/esm bench.js",
+    "docs": "node ../../scripts/generate-readmes"
+  },
```

Comment:
So I think these scripts need changing to match one of the other typescript modules (eg turf-random) just above it.
- Then I think the main.js and the main.es.js can be deleted and the `@std/esm` part of the package.json

Reply:
Ok, done. I'm not quite sure why my generated index.js looks different from the one that was already in this branch. Hopefully not a problem?